### PR TITLE
CO-533 restore docs/TASKS archive reserve

### PR DESCRIPTION
**Pull request by Kbediako**
## What
- Restore the `docs/TASKS.md` archive reserve path by treating the archive index as structural metadata rather than adjacent task snapshot content.
- Add fail-closed diagnostics for terminal-looking snapshots blocked by nonterminal `tasks/index.json` status.
- Archive the verified completed 0933/0934 snapshots and register the CO-533 packet/mirrors.

## Why
Current main could not restore the configured reserve target: `docs/TASKS.md` remained at 448 lines after the supported archive fallback, leaving ordinary docs-first packet PRs too close to the 450 hard cap.

## Validation
- `npx vitest run tests/tasks-archive.spec.ts`
- `npm run docs:archive-tasks -- --dry-run`
- `node scripts/spec-guard.mjs --dry-run`
- `npm run docs:check`
- `npm run build`
- `npm run lint` (passes with existing `DelegationMcpHealth.test.ts` warnings)
- `npm run test`
- `npm run repo:stewardship`
- `node scripts/diff-budget.mjs`
- `git diff --check`
- Manifest-backed standalone review: `review_verdict=clean`, `finding_count=0`

## Notes
`npm run docs:freshness` still fails on inherited repo-wide stale-doc baseline debt already routed to CO-522; CO-533 packet paths are registered and `missing_registry=0`.

## Summary by CodeRabbit

* **Bug Fixes**
  * Improved archive parsing to prevent structural archive markers being treated as task snapshots and enhanced diagnostics for blocked archive operations.

* **Tests**
  * Added validation tests covering archive edge cases and line‑budget behaviour.

* **Documentation**
  * Added PRD, technical spec, action plan, task spec, checklist and archive snapshot documentation for the CO‑533 work.

* **Chores**
  * Updated task index and docs freshness registry entries to reflect new items, statuses and archive records.

[Review Change Stack](https://app.coderabbit.ai/change-stack/Kbediako/CO/pull/804)

### CODE REVIEW

File: .agent/task/linear-b8b18767-1262-442f-84f3-f41b6f758d42.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- [x] `npm run repo:stewardship`. Evidence: 6475 tracked files, 0 action-required.
+- [x] `node scripts/diff-budget.mjs`. Evidence: working-tree scope 10/25 files and 488/1200 lines.
+- [x] `git diff --check`. Evidence: no whitespace errors.
+- [x] Same-issue child-lane regression coverage completion. Evidence: `.runs/linear-b8b18767-1262-442f-84f3-f41b6f758d42-regression-coverage/cli/2026-05-13T18-18-12-011Z-e69db451/manifest.json` completed successfully with no patch required.
+- [x] Standalone review and elegance pass complete. Evidence: `review/telemetry.json` reports `review_verdict=clean` and `finding_count=0`; `out/linear-b8b18767-1262-442f-84f3-f41b6f758d42/manual/elegance-review.md`.
+- [ ] PR checks, feedback sweep, and ready-review drain clean before handoff. Evidence: pending.
+
+## Handoff Status
+- [ ] PR opened and attached to CO-533. Evidence: pending.
+- [ ] Linear workpad refreshed with final validation and review status. Evidence: pending.
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply .agent/task/linear-b8b18767-1262-442f-84f3-f41b6f758d42.md:43]: @coderabbitai Fixed in de020ec883: marked PR #804 opened/attached in both checklist mirrors and reran spec-guard, docs:check, and git diff --check. Please re-review/resolve.

File: tasks/index.json [hunk 2]
```diff
@@ -18033,31 +18143,31 @@
...
+      "status": "completed",
       "gate": {
         "phase": "orchestrator-experience-jsonl-repair-planning",
         "status": "approved",
         "log": ".runs/0933-orchestrator-experience-jsonl-repair/cli/2026-01-05T18-26-16-547Z-91ff2f7d/manifest.json",
         "run_id": "2026-01-05T18-26-16-547Z-91ff2f7d"
       },
       "created_at": "2026-01-05",
-      "completed_at": null
+      "completed_at": "2026-01-05"
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply tasks/index.json:18161]: Fixed in 8379733f63: reclassified the verified completed 0933/0934 packet registry rows as archived historical metadata. Validation rerun: npm run docs:freshness now has missing_registry=0 and terminal_lifecycle=0; node scripts/spec-guard.mjs --dry-run, npm run docs:archive-tasks -- --dry-run, npm run docs:check, and git diff --check also passed.
